tests/llmobs/test_integrations_utils.py

```diff
@@ -0,0 +1,113 @@
+from ddtrace.llmobs._integrations.utils import _extract_chat_template_from_instructions
+
+
+class TestExtractChatTemplateFromInstructions:
```

Comment:
nit: we don't actually need this test class at all, all the methods can just be module-level functions